hooks/post/30-upgrade.rb

```diff
@@ -80,6 +87,7 @@ def fail_and_exit(message)
     upgrade_step :import_puppet_modules, :long_running => true
     upgrade_step :import_subscriptions, :long_running => true
     upgrade_step :remove_elasticsearch
+    upgrade_set :remove_docker_v1_content, :long_running => true
```

Comment:
@komidore64, sounds good.  Thanks!  For consistency, I would prefer it in the post as well.   I assume that it will be tested before merged.  :)